cmake/pkg-config-template.pc.in

```diff
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-includedir=${prefix}/include
-libdir=${exec_prefix}/lib
+includedir=@gRPC_INSTALL_INCLUDEDIR@
```

Comment:
Does this need to be like `${prefix}/$gRPC_INSTALL_INCLUDEDIR`?

Reply:
Thank you; after reviewing and testing, I agree. I will force-push to this PR.